pages/dug_12/4-shameless-plug.md

```diff
@@ -0,0 +1,619 @@
+</center>
+
+<!--
+
+Q1 2026
+
+-->
+
+---
+
+<center>
+
+### PCI USB controller that enables device certification for Qubes OS
+</center>
+<!--
+
+Q1 2026 ?
+
+-->
+
+---
+layout: cover
+background: /intro.png
+class: text-center
+---
+
+# Dasharo Supported Hardware
+
+---
+
+<center><img src="/@fs/repo/img/dug_12/shop_asrock.png" width="600"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/product/asrock-spc741d8-2l2t-bcm-dasharo-pro-full-build/](https://shop.3mdeb.com/product/asrock-spc741d8-2l2t-bcm-dasharo-pro-full-build/)
+</center>
+
+---
+
+<center><img src="/@fs/repo/img/dug_12/shop_gigabyte-mz33.png" width="500"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/product/full-build-gigabyte-mz33-ar1-with-dasharo-corebootuefi-pro-package-for-servers/](https://shop.3mdeb.com/product/full-build-gigabyte-mz33-ar1-with-dasharo-corebootuefi-pro-package-for-servers/)
+</center>
+
+---
+
+<center><img src="/@fs/repo/img/dug_9/dell_optiplex.png" width="450"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/product-category/dasharo-supported-hardware/](https://shop.3mdeb.com/product-category/dasharo-supported-hardware/)
+</center>
+
+<!--
+
+Dasharo (coreboot+UEFI) v0.1.1 is available.
+
+This is old Ivy Bridge hardware, but still capable and cheap.
+What is important there are multiple interesting functions available like:
+- ME Neutering - means essesntially cutting off "unneeded" pieces of ME
+- Possible TrenchBoot aka AEM for Qubes OS.
+- Possible SMI Transfer Monitor.
+- UEFI Capsule Update integration
+- New Dasharo Measured Boot which cover correctly coreboot and EDKII/UEFI payload.
+- NVMe disks support through adapter.
+- Qubes OS Certification - Unfortunately not possible due to end of life on Intel side, Intel do not provide microcode fixes, biggest value for that hardware is potential of TrenchBoot.
+
+-->
+
+---
+
+<center><img src="/@fs/repo/img/dug_11/odroid.png" width="700"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/product-category/dasharo-supported-hardware/](https://shop.3mdeb.com/product-category/dasharo-supported-hardware/)
+</center>
+
+<!--
+
+No longer under pre-order. Dasharo (coreboot+UEFI) v0.9.0 was released.
+
+Current release is minimalistic but we already planning new version which will be closer in functionality to closed source firmware.
+
+The goal of this platform is to serve as base line for hardware training which
+we provide as part of various conferences, as well as directly to customers. It
+will be also very useful for students who want to get through upcoming OST2
+UEFI Secure Boot course.
+
+-->
+
+---
+
+<center><img src="/@fs/repo/img/dug_9/novacustom_laptops.png" width="900"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/product-category/laptops/](https://shop.3mdeb.com/product-category/laptops/)
+</center>
+
+<!--
+
+We also have referral links to Novacustom. Buying through our website you
```

Comment:
@Stanislaw-bnk 
It's similar to Odroids here. We need to update the comment if we are to leave products in the presentation that we have nothing to say about today.